scripts/Muon/GUI/Common/dock/dockable_tabs.py

```diff
@@ -90,6 +99,13 @@ def detach_tab(self, index, point):
         # Create a reference to maintain access to the detached tab
         self.detachedTabs[name] = detached_tab
 
+    def determine_insert_position(self, name):
```

Comment:
Do you think this could do with a unit test? I'm not sure if it's too straightforward to require it

Reply:
it would probably make sense to look at unit tests as part of the refactoring #24678